native/src/main/java/io/ballerina/stdlib/azure/functions/Constants.java

```diff
@@ -22,49 +22,55 @@
  * {@code Constants} contains the public constants to be used.
  */
 public interface Constants {
-    String SLASH = "/";
+ String SLASH = "/";
 
-    String PACKAGE_ORG = "ballerinax";
-    String PACKAGE_NAME = "azure_functions";
-    String HTTP_PACKAGE_ORG = "ballerina";
-    String HTTP_PACKAGE_NAME = "http";
```

Comment:
Check if the line spacing is correct and the IDE is up to date with styling guidelines provided by wso2.